app/models/miq_report/formatters/graph.rb

```diff
@@ -11,6 +11,18 @@ def graph_options(options = nil)
 
   def to_chart(theme = nil, show_title = false, graph_options = nil)
     ManageIQ::Reporting::Formatter::ReportRenderer.render(ManageIQ::Reporting::Charting.format) do |e|
+      col_formats.each_with_index do |format, i|
+        next unless [:bytes_human_precision_2, :kilobytes_human, :megabytes_human, :megabytes_human_precision_2, :gigabytes_human, :gigabytes_human_precision_2].include?(format)
+
+        final_unit = 1
+        table.data.each do |d|
+          unit = [1.terabytes, 1.gigabytes, 1.megabytes, 1.kilobytes].detect { |u| d[i] / u > 10 }
+          final_unit = unit if unit > final_unit
+        end
+        table.data.each do |d|
```

Comment:
Do we have to be concerned that this is modifying the underlying table data directly?  What happens if you try to re-render the report?  (Not sure this is a real case, but it gets me wondering if to_chart is destructive)

Reply:
Not sure how to cause trouble with redraw, but at least making the chart full-screen works.